mysite/w6/beanstalk.py

```diff
@@ -6,6 +6,9 @@
 TEN_K = 1
```

Comment:
optional: change these stupid-ass names to BEANSTACKED and SUPER_BEANSTACKED

Reply:
Think these can just straight up be deleted now actually